src/wallet/coinselection.h

```diff
@@ -254,16 +254,17 @@ struct SelectionResult
 private:
     /** Set of inputs selected by the algorithm to use in the transaction */
     std::set<COutput> m_selected_inputs;
-    /** The target the algorithm selected for. Note that this may not be equal to the recipient amount as it can include non-input fees */
-    const CAmount m_target;
     /** Whether the input values for calculations should be the effective value (true) or normal value (false) */
     bool m_use_effective{false};
     /** The computed waste */
     std::optional<CAmount> m_waste;
-    /** The algorithm used to produce this result */
-    SelectionAlgorithm m_algo;
```

Comment:
why is `m_algo` added as private and then moved here to public? wouldn't it be better to move `m_target` and add `m_algo` in the first commit?

Reply:
Indeed, I forgot to squash those separately when implementing this. Will move if I have to re-touch.